src/strings/PowerFxResources.en-US.resx

```diff
@@ -6534,13 +6534,17 @@
     <comment>Error Message</comment>
   </data>
   <data name="ErrUDF_MissingColon" xml:space="preserve">
-    <value>User Defined Function is missing an ':' here</value>
+    <value>User Defined Function is missing a ':' here</value>
```

Comment:
nit: Probably, `Function definition is missing ...` for these, UDF is the feature name but feels kind of awkward in an error message

Reply:
Removed these messages entirely.